docs/decisions/0018-units-as-containers.rst

```diff
@@ -0,0 +1,37 @@
+18. Modeling Units as a Concrete Implementation of the Container Capability
+=======================================================================
+
+Context
+-------
+
+The container capability is a generalized capability to hold different types of content. This decision focuses on modeling units as a concrete implementation of the container capability.
+
+Decisions
+---------
+
+All decisions from `0017-generalized-containers.rst`_ are still valid so that this decisions will build on top of them.
+
+.. _`0017-generalized-containers.rst`: 0017-generalized-containers.rst
+
+1. Units as Containers
+=======================
+
+- A unit is a concrete type of container that holds components.
+- A unit is a container, making it also a publishable entity.
+- A unit application will offer shared definitions for use by other unit subtypes.
+
+1. Unit Types and Content Constraints
+======================================
+
+- Units can only hold components as their members but will not enforce this restriction at the model level.
+- Content restrictions for units are implemented at the app layer, allowing units to limit their members to components.
+
+1. Unit Members and Relationships
+==================================
+
+- The members of a unit can only be components.
```

Comment:
Should these decisions be part of this ADR as well?

- Unit members can exist as standalone items outside the unit.
- Members should indicate which unit they belong to.
- Members can be duplicated from units but keeping original references.